collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -519,8 +520,9 @@ private void loadForm() {
             formLoaderTask = (FormLoaderTask) data;
         } else if (data == null) {
             if (!newForm) {
-                if (getFormController() != null) {
```

Comment:
This is the one place where it seemed like we could pass a `null` to `formLoaded` as we are checking the singleton field isn't `null` and then calling the method again. Assigning it here means that we only deal with the one value (`null` or not).